charts/model-engine/templates/celery_autoscaler_stateful_set.yaml

```diff
@@ -0,0 +1,98 @@
+{{- if .Values.celery_autoscaler.enabled }}
+{{- $env := .Values.context }}
+{{- $tag := .Values.tag }}
+{{- $service_identifier := .Values.serviceIdentifier }}
+{{- $num_shards := ternary 30 3 (eq .Values.context "prod") }}
+{{- range $base_app := tuple "celery-autoscaler-elasticache" "celery-autoscaler-sqs" }}
+{{- $app := ternary (printf "%s-%s" $base_app $service_identifier) $base_app (not (empty $service_identifier)) }}
+{{- $broker_name := ternary "redis-elasticache-message-broker-master" "sqs-message-broker-master" (eq $base_app "celery-autoscaler-elasticache") }}
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app: {{ $app }}
+    team: infra
```

Comment:
These tags shouldn't be hardcoded, they're internal.

Reply:
see other deployments. something like
```
    {{- include "modelEngine.selectorLabels.celeryAutoScaler" . | nindent 4 }}
    {{- include "modelEngine.labels" . | nindent 4 }}
    tags.datadoghq.com/service: {{ include "modelEngine.celeryAutoScalerName" . }}
```
should

Reply:
yeah i think that file should parametrized tags. don't see why balloon deployment needs a different `product` tag